mcchris/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		
+		<title>mcchris</title>
+
+		<style>
+			body {
+				background-image: url(./img/bg.jpg);
+				color:white;
+				font-family: monospace;
+				margin: 6rem 1rem;
+				font-size: 120%;
+			}
+			h1, h2, h3 {
+				text-align: center;
+			}
+			.flex {
+				display: flex;
+				flex-wrap: wrap;
+				justify-content: space-evenly;
+				margin: 3rem 0;
+			}
+			a img {
+				height: 6rem;
+				width: 6rem;
+				margin: .5rem 0;
+				background-color: white;
+			}
+			a {
+
+			}
+		</style>
```

Comment:
Why does this page have inline styles, but the other page is an external stylesheet? And why is there an empty `a` tag declaration at the bottom?

Reply:
first of all, this page is the most incomplete

i was trying to scaffold a modal that would pop up over a current production page, driving visitors to buy a new product

but i dont know how to do modals without javascript

Reply:
If it's incomplete, I should not have been tagged for review. You can create a popup effect using the focus attribute in css. A quick google search for "css modal" will turn up results on the first page.